src/bench/bench.cpp

```diff
@@ -41,15 +41,37 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 } // namespace
 
+std::map<std::string, benchmark::PriorityLevel> map_priority_level = {
+    std::make_pair("high", benchmark::PriorityLevel::HIGH),
+    std::make_pair("medium", benchmark::PriorityLevel::MEDIUM),
+    std::make_pair("low", benchmark::PriorityLevel::LOW),
+    std::make_pair("all", benchmark::PriorityLevel::ALL)
+};
+
+benchmark::PriorityLevel benchmark::StringToPriority(const std::string& str)
+{
+    auto it = map_priority_level.find(str);
+    if (it == map_priority_level.end()) throw std::runtime_error(strprintf("Unknown priority level %s", str));
+    return it->second;
+}
+
+std::string benchmark::PriorityToString(PriorityLevel in_level)
+{
+    for (const auto& [name, level] : map_priority_level) {
+        if (level == in_level) return name;
+    }
+    assert(false); // out-of-sync `map_priority_level`
```

Comment:
requires `#include <cassert>`

You could also use `assert(!"<message>")` as in: https://github.com/bitcoin/bitcoin/blob/9fcdb9f3a044330d3d7515fa35709102c98534d2/src/net_processing.cpp#L2145